cluster-scope/base/core/namespaces/ushift-dev/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/project-admin-rolebindings/octo-ushift-dev
+    - ../../../../components/limitranges/default
```

Comment:
Can you please add:

```
    - ../../../../components/resourcequotas/x-small
```

So the quota is enforced, please? :slightly_smiling_face: